src/SwaggerProvider.DesignTime/v3/DefinitionCompiler.fs

```diff
@@ -411,7 +411,13 @@ type DefinitionCompiler(schema: OpenApiDocument, provideNullable) as this =
                 | "string", "binary" // for `application/octet-stream` request body
                 | "file", _ -> // for `multipart/form-data` : https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#considerations-for-file-uploads
                     typeof<IO.Stream>
+                #if NET6_0_OR_GREATER
+                // According to https://swagger.io/docs/specification/data-models/data-types/#string
+                | "string", "date" -> typeof<DateOnly>
+                #endif
+                #if NETSTANDARD2_0
```

Comment:
```suggestion
                #elif NETSTANDARD2_0
```
As far as I remember there is something like `#elif` or `#elseif`

Reply:
https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/compiler-directives#preprocessor-directives

Reply:
Fantomas is not happy with the current formatting `./build.sh -t Format` to fix

Reply:
@sergey-tihon it has some weird behavior in actions and on local machine. Probably need to check later.
For now, i targeted fantomas on this particular file
![image](https://user-images.githubusercontent.com/46487286/196460936-f61092f9-4136-43cf-99f7-3dc10151540f.png)

Reply:
Is it possible that you may use a different version of Fantomas from your IDE?